<ui version="4.0" >
 <class>LessonEdit</class>
 <widget class="QWidget" name="LessonEdit" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <widget class="QTabWidget" name="tabWidget" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>5</hsizetype>
       <vsizetype>7</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabBase" >
      <attribute name="title" >
       <string>Base</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QLabel" name="label" >
           <property name="text" >
            <string>Class:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cboxWordClass" />
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="gboxForeignLang" >
         <property name="title" >
          <string>gboxForeignLang</string>
         </property>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item row="1" column="0" colspan="2" >
           <widget class="QFrame" name="frmForeignExt" >
            <property name="focusPolicy" >
             <enum>Qt::StrongFocus</enum>
            </property>
            <property name="frameShape" >
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow" >
             <enum>QFrame::Plain</enum>
            </property>
            <property name="lineWidth" >
             <number>0</number>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QFrame" name="frmForeignGender" >
            <property name="frameShape" >
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow" >
             <enum>QFrame::Plain</enum>
            </property>
            <layout class="QHBoxLayout" >
             <property name="margin" >
              <number>0</number>
             </property>
             <property name="spacing" >
              <number>6</number>
             </property>
             <item>
              <widget class="QLabel" name="label_2" >
               <property name="text" >
                <string>Gender:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="cboxForeignGender" />
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="0" >
           <widget class="QLineEdit" name="leForeignLang" />
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gboxNativeLang" >
         <property name="title" >
          <string>gboxNativeLang</string>
         </property>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item row="1" column="0" colspan="2" >
           <widget class="QFrame" name="frmNativeExt" >
            <property name="frameShape" >
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow" >
             <enum>QFrame::Plain</enum>
            </property>
            <property name="lineWidth" >
             <number>0</number>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QFrame" name="frmNativeGender" >
            <property name="frameShape" >
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow" >
             <enum>QFrame::Plain</enum>
            </property>
            <layout class="QHBoxLayout" >
             <property name="margin" >
              <number>0</number>
             </property>
             <property name="spacing" >
              <number>6</number>
             </property>
             <item>
              <widget class="QLabel" name="label_3" >
               <property name="text" >
                <string>Gender:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="cboxNativeGender" />
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="0" >
           <widget class="QLineEdit" name="leNativeLang" />
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabDetails" >
      <attribute name="title" >
       <string>Details</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QFrame" name="frame" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>0</hsizetype>
       <vsizetype>5</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pbSave" >
        <property name="text" >
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbDelete" >
        <property name="text" >
         <string>Delete</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbNew" >
        <property name="text" >
         <string>New</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2" >
    <widget class="QTableWidget" name="twEntries" >
     <property name="editTriggers" >
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors" >
      <bool>true</bool>
     </property>
     <property name="selectionMode" >
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior" >
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="showGrid" >
      <bool>false</bool>
     </property>
     <property name="gridStyle" >
      <enum>Qt::NoPen</enum>
     </property>
     <property name="sortingEnabled" >
      <bool>false</bool>
     </property>
     <property name="columnCount" >
      <number>2</number>
     </property>
     <column/>
     <column/>
    </widget>
   </item>
  </layout>
  <action name="actionNew" >
   <property name="icon" >
    <iconset resource="../res/dvt.qrc" >:/dvt/dvt/entryNew.png</iconset>
   </property>
   <property name="text" >
    <string>New entry</string>
   </property>
  </action>
  <action name="actionDelete" >
   <property name="icon" >
    <iconset resource="../res/dvt.qrc" >:/dvt/dvt/entryDelete.png</iconset>
   </property>
   <property name="text" >
    <string>Delete entry</string>
   </property>
  </action>
  <action name="actionSave" >
   <property name="icon" >
    <iconset resource="../res/dvt.qrc" >:/dvt/dvt/lessonSave.png</iconset>
   </property>
   <property name="text" >
    <string>Save lesson</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../res/dvt.qrc" />
 </resources>
 <connections/>
</ui>
